crates/trigger/src/cli.rs

```diff
@@ -99,7 +99,7 @@ where
     pub help_args_only: bool,
 
     /// Load the application from the registry.
-    #[clap(long = "from-registry")]
+    #[clap(long = "app-from-registry")]
```

Comment:
Does the fix need to change the name of the flag as well?

Reply:
Not necessary for the fix however it is confusing to see `--from-registry` appear twice in the unified help output.

Reply:
It should probably be hidden in the trigger options.  It's not something a user should be able to pass explicitly.  (Eventually we will want to remove it entirely of course.)

Reply:
Good point!

Reply:
Not sure the fix would be replacing _one_ of them though.
It would appear that there are now two different flags with different names?

Reply:
Yeah, i changed my mind. Back to one flag but hid the trigger one.